vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -57,6 +57,9 @@ var (
 	ctrPodNameLabel     = flag.String("container-pod-name-label", "pod_name", `Label name to look for container pod names`)
 	ctrNameLabel        = flag.String("container-name-label", "name", `Label name to look for container names`)
 	vpaObjectNamespace  = flag.String("vpa-object-namespace", apiv1.NamespaceAll, "Namespace to search for VPA objects and pod stats. Empty means all namespaces will be used.")
+
+	// list of post processors flag
+	postProcessorCapping = flag.Bool("post-processor-capping", true, "Enable 'capping' post processor: apply minAllowed/maxAllowed range for the recommendation")
```

Comment:
Do we expect a situation where one would tun off capping? If no then I think we shouldn't allow doing something that we expect might only give bad results

Reply:
I would say that for the moment, there is no valid reason for unplugging capping.
With post-processors being introduced, we can have at some point use another post processor that would do capping with a different algorithm... at that time only we will have a valid reason for activating the other post-processor and removing that one.

I am removing the flag. We will re-introduce it later if it makes sense.